src/components/molecules/UiAlert/UiAlert.vue

```diff
@@ -2,7 +2,7 @@
   <div
     class="ui-alert"
     :class="rootClassModifier"
-    role="alert"
+    v-bind="alertAttrs"
```

Comment:
The computer says NO :( 
Will UiAlert with `type === 'error'` always have these `role="alert"` and `aria-live="polite"`? What about the case when I use it just like an "ornament" to mark something with this style? 
IMO it should be added only where it is needed.

Reply:
I moved alertAttrs to props. Each use can use default or his/her